src/middlewared/middlewared/plugins/gluster_linux/volume.py

```diff
@@ -416,7 +414,7 @@ async def replacebrick(self, data):
         new_brick = new['peer_name'] + ':' + new['peer_path']
 
         method = volume.bricks.replace_commit
-        options = {'args': (data.pop('name'), src_brick, new_brick), 'kwargs': data}
+        options = {'args': (data.pop('name'), src_brick, new_brick), 'kwargs': {'force': True}}
```

Comment:
Do we need to specify it if it's only supposed to be `true` ?

Reply:
I can make a comment